src/axolotl/loaders/processor.py

```diff
@@ -13,12 +13,41 @@
 LOG = get_logger(__name__)
 
 
+def _load_glm4v_processor(cfg: DictDefault, tokenizer: PreTrainedTokenizerBase):
+    """
+    Load GLM4V/GLM4V_MoE processor manually.
+
+    The model's preprocessor_config.json has an incorrect image_processor_type
```

Comment:
I'm wondering if it's even wrong https://github.com/huggingface/transformers/blob/main/src/transformers/models/glm46v/modeling_glm46v.py

This is a new addition last month

Reply:
https://github.com/huggingface/transformers/blob/dc06f2dd2e3b7a4ce3f49392ac405307a9634355/src/transformers/models/glm46v/image_processing_glm46v.py#L99